pkg/api/v1beta1/types.go

```diff
@@ -122,6 +122,8 @@ type PersistentVolumeSource struct {
 	// GCEPersistentDisk represents a GCE Disk resource that is attached to a
 	// kubelet's host machine and then exposed to the pod.
 	GCEPersistentDisk *GCEPersistentDiskVolumeSource `json:"persistentDisk" description:"GCE disk resource provisioned by an admin"`
+	// NFS represents an NFS mount on the host that persists beyond a pod's lifetime
+	NFS *NFSVolumeSource `json:"nfs" description:"NFS volume that will be mounted in the host machine "`
```

Comment:
But we want NFS for real, right?  Why not just send that as a PR now, and then you can depend on it (and this one gets a smidge smaller).